### DISCUSSION TITLE
Is this library is FIPS enabled?

### DISCUSSION THREAD

**Discussion by chetanggg666** [ProtonMail/go-crypto#197] [General]
Is this library is FIPS enabled?

**Comment by chetanggg666** [comment]
@lubux

**Reply by lubux** [reply]
The word enabled is a bit unclear, given the following definitions:

- FIPS compliant: using only cryptographic algorithms allowed by NIST
- FIPS certified: evaluated and confirmed to be compliant

`go-crypto` is not FIPS certified but might be FIPS compliant depending on the algorithms that are used. However, the library does not have a configuration option to ensure that only compliant algorithms are used.
